src/StatusQ/Components/StatusMemberListItem.qml

```diff
@@ -0,0 +1,64 @@
+import QtQuick 2.0
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+
+
+StatusListItem {
+    id: root
+
+    property string nickName: ""
+    property string userName: ""
+    property string keyChat: ""
+    property bool isMutualContact: false
+    property var trustIndicator: StatusMemberListItem.TrustedType.None
+    property string imageSource: ""
+    property bool isOnline: false
+
+    enum TrustedType {
+        None, //0
+        Verified, //1
+        Untrustworthy //2
+    }
+
+    // Subtitle composition:
+    function printSubtitile() {
+        var print = ""
+        if(root.userName !== "")
+            print = "(" + root.userName + ")"
+
+        if(print !== "" && root.keyChat !== "")
+            // Composition
+            print += " • " + printShortKeyChat(root.keyChat)
+
+        else if(root.keyChat !== "")
+            print = printShortKeyChat(root.keyChat)
+
+        return print
+    }
+
+    // Short keychat composition:
+    function printShortKeyChat(keyChat) {
+        return keyChat.substring(0, 5) + "..." + keyChat.substring(keyChat.length - 3)
+    }
+
+    // root object settings:
+    title:  root.nickName
+    titleIcon1Visible: root.isMutualContact
+    titleIcon2Visible: root.trustIndicator !== StatusMemberListItem.TrustedType.None
+    subTitle: printSubtitile()
+    subTitlePixelSize: 10
+    icon.isLetterIdenticon: root.imageSource === ""
+    image.source: root.imageSource
+    image.isIdenticon: root.imageSource !== ""
```

Comment:
Hm.. so this doesn't look right to me. Why do we assume that we're dealing with an identicon, when `imageSource != ""` ?

Keep in mind that a normal profile picture is also just an image source.

So I guess what needs to happen here instead is that we remove the `imageSource` introduced n `StatusMemberListItem` and simply rely on the already existing `image.[...]` properties which allow consumers to set `isIdenticon` etc respectively.

Am I missing something here otherwise?

Reply:
Ok, so I'll only set the following to always have, at least, a letter as default:
```icon.isLetterIdenticon: !root.image.source.toString()```

Reply:
One possibility although, we might as well default it to `false` because I don't think members will ever have a letter identicon. Those are used for channels in communities.

I think the default of `isLetterIdenticon` is even `false`. If not, you can default it to `false` inside of this component.

Reply:
I don't think it should be false as default. I understand from the design that each member in the list should have or an image/identicon or a letter identifying him/her. If it is set to false, the StatusMemberList item didn't display the circle with the member identification and would be only its nickName, userName and/or chatKey, and what would happend with the connect/disconnect indicator? 

From the design, the possibilities I understand are, at least, showing a letter: 
![image](https://user-images.githubusercontent.com/97019400/151130948-2255eff8-69bf-43ef-a797-26ce320af584.png)

Reply:
> I understand from the design that each member in the list should have or an image/identicon or a letter identifying him/her.

Ah, this is new and wasn't the case before. Users could only have either an identicon or a profile picture. I see letter identicons being used for accounts the first time here.

So then, I'd agree with your and probably would even say that it should default to `true` **however**, I wonder how this is done on a model/data basis. Right now every account has an `imageSource` because every account has an identicon by default. 

So if we go down that road, we probably need to discuss (with backend possibly) how we detect that indeed a letteridenticon should be rendered and not the normal identicon

Reply:
> So if we go down that road, we probably need to discuss (with backend possibly) how we detect that indeed a letteridenticon should be rendered and not the normal identicon

Yes, maybe a discussion with the backend is needed or there is already a task to do it. Think we are now doing just the qml components while we wait for `base_bc` to be integrated to master.

Reply:
> So then, I'd agree with you and probably would even say that it should default to `true` **however**, I wonder how this is done on a model/data basis. Right now every account has an `imageSource` because every account has an identicon by default.

We cannot set it true as default because of the way the `StatusSmartIdenticon` is implemented as setting it to true will force the letter option always. 

See the piece of code where `StatusSmartIdenticon` decides about the **sourceComponent** to be loaded:

```
sourceComponent: statusSmartIdenticon.icon.isLetterIdenticon ? letterIdenticon :
                 !!statusSmartIdenticon.image.source.toString() ? roundedImage :
                 !!statusSmartIdenticon.icon.name.toString() ? roundedIcon : letterIdenticon```

Reply:
Ah good find! That let's stick with the expression you've proposed. We might want to adjust `StatusSmartIdenticon` in the future to be a bit more flexible on that regard